plugins/postcss-design-tokens/src/data-formats/style-dictionary/v3/group.ts

```diff
@@ -7,12 +7,20 @@ export type StyleDictionaryV3TokenGroup = {
 }
 
 function extractTokens(node: StyleDictionaryV3TokenGroup, path: Array<string>, filePath: string): Map<string, StyleDictionaryV3TokenValue> {
-	const result: Map<string,StyleDictionaryV3TokenValue> = new Map();
+	const result: Map<string, StyleDictionaryV3TokenValue> = new Map();
 	for (const key in node) {
 		if (Object.hasOwnProperty.call(node, key)) {
+			if (
+				typeof node[key] !== 'object' ||
+				Array.isArray(node[key]) &&
+				node[key] === null
+			) {
+				throw new Error(`Parsing error at "${[...path, key].join('.')}"`);
+			}
+
 			const child = Object(node[key]);
 			if (!child) {
-				throw new Error('Parsing error');
+				throw new Error(`Parsing error at "${[...path, key].join('.')}"`);
```

Comment:
@Antonio-Laguna we can also choose to ignore these issues.
This will give users more freedom.

At the moment we require that each object has a `"value"` property.
If this is missing we emit a `Parsing error`.

Thoughts?

Reply:
What would be an usecase for not having a `value` property? If there's none, I think we should just error since you're probably missing something?

Reply:
Users might stuff extra meta data into the same JSON file.
Since there is no specification this is "allowed".

Starting with an error makes more sense to me.
If users need to this to be more forgiving they can ask us to change it or make it configurable.

Reply:
Yep, agreed